README.md

```diff
@@ -1,4 +1,59 @@
-# Private Identity Server
+# Private Identity Server (PIS)
 
 - `/client` - Create React App front-end for identity service
 - `/server` - Flask private identity server
+
+## Legend
+* May not be necessary for MVP [color=#a01111]
+* Needs more thought/conversation [color=#0011a1]
+## Requirements
+* Landing page explaining the project
+* Single Sign On integration with Multiple Providers
+    * Providers:
+        * Google
+        * Facebook
+        * Twitter
+        * Linkedin
+        * Banks (Capital One, etc.)
+        * Etc.
+    * Should allow for ease of connecting all social accounts into one PIS account
+    * No data other than the credentials and unique identifiers for accessing the corresponding APIs should be stored in the PIS DB
+* Vending Credentials for Identity Verification
+    * Accomplished by Aries, in the long run. See https://hackmd.io/qeso7QTVRXqXf4uL6cbdgQ?sync=&type= for more info
+    * Credential should be specific to the third-party organization requesting the data
+        * i.e. if Verily wants access to birthdate and sex information, they will receive a token that is specific to their service for querying information about this user. This would be different from the token generated for another COV19 screening application asking for the same information from the same user.
+    * Credential should include which fields are accessible via the token in question
+    * Credential should be revokable by the user
+    * Credential should be acquired by the third-party application via a single-sign on flow where this identity server is the identity provider
+* Securely disclosing personal information to third-party applications without disclosing the identity of the person involved
+    * In-memory data retrieval from the various SSO destinations will be necessary
+    * API that will be accessible to these third-party applications to retrieve that data in real-time, authenticated by the credential
+        * List of disambiguated data fields accessible via the API, along with each source that can provide that information
+            * Some sophisticated change-management will be needed here, as this will be subject to change often. Especially given that we want to allow for this to be hosted in a decentralized fashion.
```

Comment:
Since, this is planned to be a decentralized application, is it like an instance of this PIS can be deployed by any individual/organization and they need to authenticate themselves with this PIS to get access to the data? I am actually trying to understand this. Please correct me on this if wrong.

Reply:
Great question! I updated the readme to address this elsewhere, but basically, there will be no PIS to PIS communication with this model. In essence, if user A has linked their SSO accounts to host 1, then in order for host 2 to provide data for user A, user A would have to link all their SSO accounts with host 2 as well. This is less a "decentralized app" and more "many instances of a single centralized app".

Reply:
Got it @carrollgt91. So, if I am not wrong, the rationale behind having many instances of single centralized app is make it deploy-able as an on-premise solution as well. There will an online deployed instance as well.

Reply:
Yes, I think that's a good way of understanding it.